README.md

```diff
@@ -1,93 +1,56 @@
-    envoy-users@ per the guidance in the following linked thread.
-
-Please see [this](https://groups.google.com/forum/#!topic/envoy-announce/l9zjYsnS3TY) email thread
-for information on email list usage.
-
-## Contributing
-
-Contributing to Envoy is fun and modern C++ is a lot less scary than you might think if you don't
-have prior experience. To get started:
-
-* [Contributing guide](CONTRIBUTING.md)
-* [Beginner issues](https://github.com/envoyproxy/envoy/issues?q=is%3Aopen+is%3Aissue+label%3Abeginner)
-* [Build/test quick start using docker](ci#building-and-running-tests-as-a-developer)
-* [Developer guide](DEVELOPER.md)
-* Consider installing the Envoy [development support toolchain](https://github.com/envoyproxy/envoy/blob/master/support/README.md), which helps automate parts of the development process, particularly those involving code review.
-* Please make sure that you let us know if you are working on an issue so we don't duplicate work!
-
-## Community Meeting
-
-The Envoy team meets twice per month on Tuesday, alternating between 9am PT and 5PM PT. The public
-Google calendar is here: https://goo.gl/PkDijT
-
-* Meeting minutes are [here](https://goo.gl/5Cergb)
-* Recorded videos are posted [here](https://www.youtube.com/channel/UCvqbFHwN-nwalWPjPUKpvTA/videos?view=0&sort=dd&shelf_id=1)
-
-## Security
-
-### Security Audit
-
-A third party security audit was performed by Cure53, you can see the full report [here](docs/SECURITY_AUDIT.pdf).
-
-### Reporting security vulnerabilities
-
-If you've found a vulnerability or a potential vulnerability in Envoy please let us know at
-[envoy-security](mailto:envoy-security@googlegroups.com). We'll send a confirmation
-email to acknowledge your report, and we'll send an additional email when we've identified the issue
-positively or negatively.
-
-For further details please see our complete [security release process](SECURITY.md).
+# WAF extension on Envoy proxy
+
+This repository is forked from [`envoyproxy/envoy-wasm`](https://github.com/envoyproxy/envoy-wasm), and the example WASM
+extension in the envoy-wasm repository is modified to work as a Web Application Firewall(WAF) that
+can detect SQL injection. The rules for detection are aligned with ModSecurity
+rules [942100](https://github.com/coreruleset/coreruleset/blob/v3.3/dev/rules/REQUEST-942-APPLICATION-ATTACK-SQLI.conf#L45) and [942101](https://github.com/coreruleset/coreruleset/blob/v3.3/dev/rules/REQUEST-942-APPLICATION-ATTACK-SQLI.conf#L1458), and SQL injection is detected with methods from
+[libinjection](https://github.com/client9/libinjection).
+
+## Deployment
+From the root of the repository, build static binary of envoy proxy:
+
+```bazel build -c opt //source/exe:envoy-static```
+
+Run tests for envoy to make sure the binary has been built successfully:
+
+```bazel test //test/common/common/...```
+
+The source code for the WASM extension is in `examples/wasm`. Build the WASM module with:
```

Comment:
You probably need to specify at this point that people running this command need to first have checked out libinjection repository in upper directory.

Reply:
Updated in [this commit](https://github.com/dengyijia/envoy-wasm/pull/1/commits/3570d0ceb24647fc8dbf3d43afa2e9c8fee9df90)